AutoGrade/grader.py

```diff
@@ -88,6 +88,66 @@ def run_student_tests(target_folder, total_points, timeout):
     out_file =  os.path.join(target_folder, "test-results" + ".log")
     touch(out_file)
 
+
+    """
+    # This code can be used to execute and compile code throught compile, In reference to PR "Languagespp - Temporary Record"
+
+        # Compiling
+        logger.debug("Checking code for compilation error")
+
+        # We need to bring assignment variable here.
+        cmd = assignment.language.compile_cmd
+        cmd = process_cmd(cmd, mainfile, timeout, basename)
+        cmd = cmd.split(" ")
```

Comment:
This file contains code in comment to process command stored in Language model to compile and execute code.